### ISSUE TITLE
Overwrite react-reader default styles

### ISSUE THREAD

**Issue by avalero** [gerhardsletten/react-reader#141]
On documentation says that the way to overwrite the styles is:

```javascript
const ownStyles = {
  ...ReactReaderStyle,
  arrow: {
    ...ReactReaderStyle.arrow,
    color: 'red'
  }
}

const App = () => {
  return (
    <div style={{ height: '100vh' }} className="myReader">
      <ReactReader
        url="https://react-reader.metabits.no/files/alice.epub"
        readerStyles={ownStyles}
      />
    </div>
  )
}
```

But `ReactReaderStyle` is a type, not a value. In order to make this I think you should export on the package `reactReaderStyles` on `src/modules/ReactReader.style.js`

Thanks for your work

**Comment by gerhardsletten** [comment]
@avalero Should probably rewrite the whole package to TS soon, but did the updated in `react-reader@1.1.0` work for you now?

**Comment by AjayManikantaG** [comment]
@gerhardsletten Could you please let me know for any workaround for now to change styles as needed?

Thanks,
Ajay

**Comment by gerhardsletten** [comment]
@AjayManikantaG Sorry, but it should be fixed in `react-reader@1.2.1` #144

**Comment by avalero** [comment]
Thanks, withreact-reader@1.2.0 it works
